.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: 🛠 Bug report
+about: Create a report to help us improve
+title: "[Bug Report] Good bug title tells us about precise symptom, not about the root cause."
+labels: "bug"
+assignees: ""
+---
+
+## Description
+<!-- A clear and concise description of what the bug is. -->
+
+## {{ cookiecutter.project_name }} version
+<!-- x.y.z -->
```

Comment:
Note that there currently isn't any released versions (not sure there ever will be, given the nature of this script)

Reply:
Let me add comments like "x.y.z or commit hash". Thanks for the comment!